### ISSUE TITLE
Issues with Bower dist

### ISSUE THREAD

**Issue by alexisabril** [bitovi/funcunit#113]
`main` target is `dist/funcunit.js`, which is missing the `browser` folder within `dist`. CJS & AMD version are also missing this folder.

**Comment by matthewp** [comment]
main should be pointing to the global, is it not?

**Comment by alexisabril** [comment]
It does actually, I'm tracking down the actual cause. Just putting a bug note here before I forget.

**Comment by matthewp** [comment]
Here's the browser folder in the commonjs build: https://github.com/bitovi/funcunit/tree/v3.0.0/dist/cjs

It does have browser, same for the amd. But it's not showing up in bower?

**Comment by matthewp** [comment]
Just tried it and you are right... very bizarre, should be there, it's in the tag.

**Comment by alexisabril** [comment]
we ignore "browser" instead of "/browser" in the bower.json, however I haven't debugged why the main dist isn't working.

**Comment by alexisabril** [comment]
Note: jQuery shouldn't be a dependency. Maybe we shouldn't distribute the non-global version via bower at all(which would remove the syn dep as well). Reason is: bower doesn't handle nested dependencies, so the version of jQuery will most likely always conflict with the version the app being tested needs.

**Comment by matthewp** [comment]
jQuery is definitely a dependency, FuncUnit doesn't work without it.  The user can override our version using resolutions: http://jaketrent.com/post/bower-resolutions/

**Comment by matthewp** [comment]
We should probably provide a way for the user to pass in their own jQuery and we create a "new funcunit" based off of that.

**Comment by alexisabril** [comment]
Correct, however jQuery 1.11.x should be built into the dist/funcunit.js in noConflict mode. This would allow you to test an app that uses jQuery or any $ lib of any version, independent of the testing framework.

**Comment by matthewp** [comment]
It's not built into the global.  It expects you to put jquery in a <script> before the FuncUnit <script>

**Comment by alexisabril** [comment]
Just noting for this issue: FuncUnit does build jQuery 1.11.x into the global: https://github.com/bitovi/funcunit/blob/master/dist/funcunit.js#L2430

We only expected jQuery to be added separately in v2.0, which was actually my personal mistake. In v2.1.0-pre, this was corrected.
